Sprint-3/1-implement-and-rewrite-tests/rewrite-tests-with-jest/package-lock.json

```diff
@@ -0,0 +1,4275 @@
+{
+  "name": "rewrite-tests-with-jest",
+  "version": "1.0.0",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {
```

Comment:
Please remove the package-lock.json you added inside the Sprint directory.
the root-level Node configuration can be used.